circleci.sh

```diff
@@ -49,16 +45,15 @@ fi
 
 posturl="https://circleci.com/api/v1.1/project/github/$githubAccountName/$projectName/envvar?circle-token=$circleCiToken"
 
-
 echo "Loading environment variables into project $githubAccountName/$projectName:"
 
-for variable in $variables;
+# Iterate through your variables in config.txt and upload them to circle ci
+# They need to be in the shape of "VARIABLE_NAME|VALUE"
+IFS="|"
+while read -r name value
 do
-    
-    set -- `echo $variable | tr '=' ' '`
-    curl -X POST --header "Content-Type: application/json" -d "{\"name\":\"$1\", \"value\":\"$2\"}" $posturl
-
-done
+curl -X POST --header "Content-Type: application/json" -d "{\"name\":\"$name\", \"value\":\"$value\"}" $posturl
+done < config.txt
```

Comment:
Sweet. That will certainly make life easier. Would it be also worth allowing the config file name to be passed as parameter? The usage on CI/CD pipelines would probably be something like:

sh circleci.sh -c $CONFIG_NAME -t $CC_TOKEN> -a $GIT_ACC_NAME -p $PROJECT_NAME

And the CONFIG_NAME would vary based on the environment:

* config.ci.txt
* config.prod.txt

Reply:
This makes sense. Will add it in the next pr.